src/components/ui/ImageGallery/ImageGallerySelector.tsx

```diff
@@ -50,6 +58,14 @@ function ImageGallerySelector({ images, onSelect, currentImageIdx }: Props) {
       )}
       <div data-fs-image-gallery-selector-elements ref={elementsRef}>
         {images.map((image, idx) => {
+          let ref
+
+          if (idx === 0) {
+            ref = firstImageRef
+          } else if (idx === images.length - 1) {
+            ref = lastImageRef
+          }
+
```

Comment:
I wonder if should we use const here for the sake of [prefer-const](https://eslint.org/docs/rules/prefer-const).

```js
          const ref =
            idx === 0
              ? firstImageRef
              : idx === images.length - 1
              ? lastImageRef
              : null
```